connection/cluster/ClusterServerClient.java

```diff
@@ -22,22 +22,65 @@
 package com.vaticle.typedb.client.connection.cluster;
 
 import com.vaticle.typedb.client.api.connection.TypeDBCredential;
+import com.vaticle.typedb.client.common.exception.TypeDBClientException;
 import com.vaticle.typedb.client.connection.TypeDBClientImpl;
+import io.grpc.ManagedChannel;
+import io.grpc.netty.GrpcSslContexts;
+import io.grpc.netty.NettyChannelBuilder;
+import io.netty.handler.ssl.SslContext;
+
+import javax.net.ssl.SSLException;
 
 class ClusterServerClient extends TypeDBClientImpl {
 
-    private final ClusterServerStub clusterServerStub;
```

Comment:
This stub is a code smell that can be removed. We already have a stub in `TypeDBClientImpl` which we can use.